README.md

```diff
@@ -13,7 +21,7 @@ funcpack pack ./
 
 You can then test locally using the CLI tool: `func run <myfunc>`
 
-When uploading your files, you need to include your `package.json`, but you don't need your `node_modules` directory.
+When uploading your files, you need to include the single `.funcpack` directory (in the Functions App root), but you don't need your `node_modules` directory for each Function.
```

Comment:
I totally agree - at least you should pull the common modules up to the function app. However none of the examples or docs demonstrate that. I'll chane the wording to cover all cases

Why did you mention package.json?